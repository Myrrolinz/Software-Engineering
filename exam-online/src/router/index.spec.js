import { shallowMount } from '@vue/test-utils';
import index from './index';





describe('<index/>', () => {
	const wrapper = shallowMount(index);

	// 快照测试
	it('snapshot测试', () => {
		const wrapper2 = shallowMount(index);
		expect(wrapper2).toMatchSnapshot()
		wrapper2.destroy()
    })

	
  });