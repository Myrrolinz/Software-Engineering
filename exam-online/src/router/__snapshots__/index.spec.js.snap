// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<index/> snapshot测试 1`] = `
VueWrapper {
  "_emitted": Object {},
  "_emittedByOrder": Array [],
  "isFunctionalComponent": undefined,
}
`;
